src/Instrumentation/PDO/src/SqlServerPropagator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Instrumentation\PDO;
+
+class SqlServerPropagator implements SqlPropagatorInterface
```

Comment:
@jerrytfleung naming nit ;)... IMHO `ContextInfoPropagator` makes more sense given the other one is called `SqlCommentPropagator`.